.env

```
@@ -2,6 +2,13 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=password
 POSTGRES_DB=ct_photos_database
 
+MINIO_ENDPOINT=minio:9000
+MINIO_ACCESS_KEY=minioadmin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ACCESS_KEY key should go before the MINIO_ENDPOINT key

(UnorderedKey)